runtime/manifest.js

```diff
@@ -214,6 +214,9 @@ class Manifest {
   findShapeByName(name) {
     return this._find(manifest => manifest._shapes.find(shape => shape.name == name));
   }
+  findRecipeByName(name) {
```

Comment:
Do we need to support multiple here, like `findParticlesByVerb` above?

Reply:
findRecipesByName? Or findRecipesByVerb?

Reply:
(the latter. Done.)

Reply:
could a recipe (in the future) have both - a name and a verb that are different?